spec/models/case_closure/offender_complaint_approval_flag_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+module CaseClosure
+  RSpec.describe OffenderComplaintApprovalFlag::ICOApprovalFlag, type: :model do
+
+    it { should validate_presence_of(:name) }
+    it { should validate_presence_of(:abbreviation) }
+    it { should validate_presence_of(:sequence_id) }
+
+    describe '.id_from_name' do
+      it 'returns the id when name exists in the database' do
+        complaint_first_approval = create :offender_ico_complaint_approval_flag, :first_approval
+        complaint_second_approval = create :offender_ico_complaint_approval_flag, :second_approval
+        complaint_not_approval_required = create :offender_ico_complaint_approval_flag, :not_approval_required
+
+        expect(OffenderComplaintApprovalFlag::ICOApprovalFlag.id_from_name(complaint_first_approval.name))
```

Comment:
Small thing but it might be easier to read if we were to store the class in a variable and refer to that rather than calling it directly multiple times. Only that the class names are so long it's a lot to look at.